### ISSUE TITLE
Test 2

### ISSUE THREAD

**Issue by ivozandhuis** [netwerk-digitaal-erfgoed/LDWizard#26]
Wrong header-line?

[Archieven en collecties bij het Streekarchief.txt](https://github.com/netwerk-digitaal-erfgoed/LDWizard/files/5067707/Archieven.en.collecties.bij.het.Streekarchief.txt)

**Comment by wouterbeek** [comment]
Indeed, the header row contains the following cells (these are also identified in LD Wizard):

- `"Bestandsnummer"`
- `"Archieftitel"`
- `"Begindatum"`
- `"Einddatum"`
- `"Aantal"`
- `"Eenheid"`
- `"url"﻿"SAGV411"` (<--- possibly a newline character is missing between these two double quote characters)
- `"Archief van de stichting Exedra"`
- `"1997"`
- `"2001"`
- `"0.5"`
- `"m"`
- `"http://zoeken.gooienvechthistorisch.nl/detail.php?id=246867090"`
